lms/static/sass/shared/_footer-edx.scss

```diff
@@ -7,17 +7,116 @@
 $edx-footer-link-color: $link-color;
 $edx-footer-bg-color: rgb(252,252,252);
 
-
 // Aggressively scoped for Drupal
 // ==============================
 // These styles are being loaded on Drupal, LMS and WordPress
 // sites so the scope has to be aggressive to override default
 // CMS styles
 footer#footer-edx-v3 {
   background: $edx-footer-bg-color;
-  padding: 20px;
+  padding: 15px 0;
   border-top: 1px solid $courseware-button-border-color;
+  font-family: "Open Sans" ;
+
+  div > div > div:nth-child(2) > div.col-lg-2.edx-col-md-2.col-md-3.col-sm-3 > ul {
```

Comment:
This is overly specific, and very brittle to changes in the HTML. What is this trying to target? It would be better to add a class that you can target on the `ul`. Note that our best practice is to avoid targeting elements as it makes it hard to avoid the rule accidentally applying to unintended elements.

Reply:
@andy-armstrong yes right removed them in the latest update.